functions/src/SQR.functions.js

```diff
@@ -83,7 +142,20 @@ exports.syncStorageToDB = functions.https.onRequest((req, res) => {
 
 
 exports.importMP3IntoSQR = functions.storage.object().onFinalize( object => {
```

Comment:
Better to name it `importFilesFromStorage`.

Reply:
ok